### DISCUSSION TITLE
completely check multiple files / entire directory?

### DISCUSSION THREAD

**Discussion by o-az** [kaleidawave/ezno#80] [Q&A] [answered]
How can I run `oxidation-compiler` to typecheck multiple files by passing a glob string / directory path?

Works:
```ts
npx oxidation-compiler@latest check src/file.tsx
```
None of these commands check all files:
```ts
npx oxidation-compiler@latest check src/*.tsx
```
```ts
npx oxidation-compiler@latest check src/
```
```ts
npx oxidation-compiler@latest check src/*
```
![CleanShot 2023-11-11 at 10 48 53@2x](https://github.com/kaleidawave/ezno/assets/23618431/be44ae4c-5687-4f07-8af1-32807191aec2)

**Comment by kaleidawave** [comment, marked as answer]
Not currently supported. Opened an issue for it here https://github.com/kaleidawave/ezno/issues/81. It sort of requires some changes to the internal checker. Once it is supported in Ezno's CLI, it should be trivial to add on the Oxc side as well.
